docs/resources/ga_address_group.md

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Global Accelerator (GA)"
+---
+
+# huaweicloud_ga_address_group
+
+Manages a GA IP address group resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "name" {}
+
+resource "huaweicloud_ga_address_group" "test" {
+  name        = var.name
+  description = "created by terraform"
+
+  ip_addresses {
+    cidr        = "192.168.1.0/24"
+    description = "test acc"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, String) Specifies the IP address group name.
+
+* `description` - (Optional, String) Specifies the description of the IP address group.
+
+* `ip_addresses` - (Optional, List) Specifies the CIDR block of the IP addresses group.
+  The [ip_addresses](#ip_address_list) structure is documented below.
+
+<a name="ip_address_list"></a>
+The `ip_addresses` block supports:
+
+* `cidr` - (Required, String) Specifies the CIDR block associated with IP address.
+
+* `description` - (Optional, String) Specifies the description of the IP address.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `status` - The status of the IP address group. The value can be one of the following:
+  + **ACTIVE**: The resource is running.
+  + **PENDING**: The status is to be determined.
+  + **ERROR**: Failed to create the resource.
+  + **DELETING**: The resource is being deleted.
```

Comment:
It seems that the status **pending** and **deleted** do not appear in the display of its status after the resource is actually created

Reply:
done